capa/engine.py

```diff
@@ -104,9 +108,9 @@ def __nonzero__(self):
 class And(Statement):
     """match if all of the children evaluate to True."""
 
-    def __init__(self, *children):
-        super(And, self).__init__()
-        self.children = list(children)
+    def __init__(self, children, description=None):
+        super(And, self).__init__(description)
```

Comment:
since `description` is a kwarg, i prefer specifying this via `description=description`

Reply:
why is it a `kward`? From [Python documentation](https://book.pythontips.com/en/latest/args_and_kwargs.html):

> **kwargs allows you to pass keyworded variable length of arguments to a function

Isn't this used to allow the function to receive an arbitrary number of parameters? This has a fixed number of parameters, but one is optional. Or has I understood something wrong? 😕

Reply:
kwarg is the name for a parameter like `foo=bar`, aka a parameter with a default value. kwargs don't have to be passed in order (but typically are) when they have their keyword provided. when a function supports arbitrary kwargs, that is, when not all of the possible keywords are pre-declared, then it accepts a positional argument `**kwargs` that collects all of these into a dict. but, thats not whats happening here. I'm just suggesting that we use the `description=` prefix so that its clear we're referencing the kwarg, not a position arg (which cannot be referenced by key, at least in py2.7).

Reply:
I think I get it now. I should used the `key=value` format when calling parameters with a default value, right? Should we also update this in other places for consistency? I am concretely thinking about the description implementation in features (which looks the same as here).